spec/support/initializers/test_prof.rb

```diff
@@ -7,6 +8,12 @@
     record.tap(&:readonly!)
   end
 
+  # marks the record as readonly
   config.alias_to :let_it_be_readonly, readonly: true
+
+  # reloads the record from the DB, with `record.reload`
   config.alias_to :let_it_be_changeable, reload: true
```

Comment:
Does this mean the record persists for all the tests and we simply reload it in between?

Reply:
Exactly

Reply:
In my experience, I feel like these always cause issues and it's probably better to use fresh data rather than re-use data. Might be an interesting QA question 🤔

Reply:
Yeah, I agree, but we already use it, I just added a comment.

I was a lukewarm towards it in the beginning but I think it's hampering our ability to detect ordering issues, or at least that's my not yet fully formed opinion after playing with `rspec --bisect`.

The reason why it exists in the first place (and why `let_it_be` in general exists in our code base) is to solve another problem: our tests being slow. We have too much setup data in some tests so a while back we started using [let_it_be](https://test-prof.evilmartians.io/#/let_it_be) to solve that issue (we already had the ordering problem TBH).

I haven't yet found the perfect compromise between slow tests and flakyness but obviously I'd rather have slow tests that can be sped up than flaky tests that pile up. 

I'm going to work more with `rspec --bisect` in my downtime and hopefully track down some other ones. I spent a long time hunting an unexplicable test failure that only happens in a particular order. I have the receipts and screenshots if you want them, it's almost a comical adventure :D